tracee-ebpf/main.go

```diff
@@ -1115,6 +1131,40 @@ func unpackBPFBundle(dir string) error {
 	return nil
 }
 
+// unpackBTFHub unpacks tailored, to the compiled eBPF object, BTF files for kernel supported by BTFHub
+func unpackBTFHub(outFilePath string, OSInfo *helpers.OSInfo) error {
+	var btfFilePath string
+
+	osId := OSInfo.GetOSReleaseFieldValue(helpers.OS_ID)
```

Comment:
It might be a problem to use this API of OSInfo in tracee container (alpine based) as we will get the wrong distro release info (https://github.com/aquasecurity/libbpfgo/issues/92)

Reply:
Let's discuss this at: https://github.com/aquasecurity/libbpfgo/issues/92#issuecomment-969078143 so we have a single point to talk about it. IIRC we already addressed this in the past, but we can discuss in that issue you opened if you are okay.

Reply:
Even if we agree to use the mock osrelease solution, this PR will have to be updated, or the tracee container would not detect the correct osrelease (as it is based on an alpine container)

Reply:
If I understood correctly what you are saying, you mean that we have to make sure the mock is done to tracee container within this PR (and not other). This way, tracee container would run by default with LIBBPFGO_OSRELEASE_FILE setting correctly having the host information. 

Is that what you meant ?

Reply:
Yes, exactly

Reply:
This still needs to be addressed.